nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/main/java/org/nuxeo/ecm/platform/comment/impl/AbstractCommentManager.java

```diff
@@ -167,4 +168,20 @@ protected void setCommentPermissions(CoreSession session, DocumentModel document
         return ancestorIds;
     }
 
+    /**
+     * Notify the event of type {@code eventType} on the given {@code commentDocumentModel}.
+     *
+     * @param session the session
+     * @param commentDocumentModel the document model of the comment
+     * @param eventType the event type to publish
+     * @implSpec This method uses internally {@link #notifyEvent(CoreSession, String, DocumentModel, DocumentModel)}
+     * @since 11.1
+     */
+    protected void notifyEvent(CoreSession session, DocumentModel commentDocumentModel, String eventType) {
+        requireNonNull(eventType);
```

Comment:
why only on this param?

Reply:
if  one of `session`,  `commentDocumentModel` is `null` the code will fail few lines after, but there is no check on  `eventType` even in the `EventImpl` it's why i added just for it to fail before i notify an event which is `null` (early check).